src/test/java/cz/startnet/utils/pgdiff/PgDiffTest.java

```diff
@@ -368,6 +368,7 @@ public void runDiff() throws FileNotFoundException, IOException {
         arguments.setAddDefaults(addDefaults);
         arguments.setIgnoreFunctionWhitespace(ignoreFunctionWhitespace);
         arguments.setIgnoreStartWith(ignoreStartWith);
+       PgDiffUtils.setUseExists(true);
```

Comment:
You might want to fix the indenting here, as well as add a separate test case where useExists is false and everything else is constant